e2e/hook_test.go

```diff
@@ -44,34 +44,6 @@ func testHook() {
 	})
 
 	It("should test hooks", func() {
-		By("waiting controller pod become ready")
```

Comment:
deleted this test because this and the following test are duplicated:
https://github.com/topolvm/topolvm/blob/v0.12.0/e2e/suite_test.go#L124-L133